examples/fewshot_protonet/test.py

```diff
@@ -0,0 +1,94 @@
+"""
+This example is about testing prototypical networks to perform N-Way-K-Shot problems.
+
+Reference:
+    Snell, J., Swersky, K. and Zemel, R., 2017.
+    Prototypical networks for few-shot learning.
+    Advances in neural information processing systems, 30.
+"""
+import argparse
+import os
+from datetime import datetime
+
+import pytorch_lightning as pl
+import torch
+from config import get_cfg_defaults
+from torch.utils.data import DataLoader
+from torchvision import transforms
+from torchvision.models import *
+
+from kale.embed.image_cnn import *
+from kale.loaddata.few_shot import NWayKShotDataset
+from kale.pipeline.fewshot_trainer import ProtoNetTrainer
+
+
+def arg_parse():
+    parser = argparse.ArgumentParser(description="Args of ProtoNet")
+    parser.add_argument("--cfg", default="configs/omniglot_resnet18_5way5shot.yaml", type=str)
+    parser.add_argument("--devices", default=1, type=int)
+    parser.add_argument("--ckpt", default=None)
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    # ---- get args ----
+    args = arg_parse()
+
+    # ---- get configurations ----
+    cfg_path = args.cfg
+    cfg = get_cfg_defaults()
+    cfg.merge_from_file(cfg_path)
+    cfg.freeze()
+
+    # ---- set model ----
+    net = eval(f"{cfg.MODEL.BACKBONE}(weights={cfg.MODEL.PRETRAIN_WEIGHTS})")
+    if cfg.MODEL.BACKBONE.startswith("resnet"):
+        net.fc = Flatten()
+    model = ProtoNetTrainer(cfg=cfg, net=net)
+
+    # ---- set data loader ----
+    transform = transforms.Compose(
```

Comment:
Double-check: is this different from `  transform = get_transform(kind="few-shot", augment=False) `?

Reply:
Changed